executor/executor.go

```diff
@@ -115,22 +115,37 @@ func (e *ShowDDLExec) Next() (*Row, error) {
 		return nil, errors.Trace(err)
 	}
 
-	info, err := inspectkv.GetDDLInfo(txn)
+	ddlInfo, err := inspectkv.GetDDLInfo(txn)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	bgInfo, err := inspectkv.GetBgDDLInfo(txn)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
 
 	rowData := []interface{}{
-		info.SchemaVer,
+		ddlInfo.SchemaVer,
+		"",
 		"",
+		bgInfo.SchemaVer,
 		"",
+		"",
+	}
+	if ddlInfo.Owner != nil {
+		rowData[1] = ddlInfo.Owner.String()
 	}
-	if info.Owner != nil {
-		rowData[1] = info.Owner.String()
+	if ddlInfo.Job != nil {
+		rowData[2] = ddlInfo.Job.String()
 	}
-	if info.Job != nil {
-		rowData[2] = info.Job.String()
+	if bgInfo.Owner != nil {
+		rowData[4] = bgInfo.Owner.String()
 	}
+	if bgInfo.Job != nil {
+		rowData[5] = bgInfo.Job.String()
+	}
+
```

Comment:
Don't use hard coded '1', '2', '4', '5', declare a variable before rowData, assign its value, then initialzie the rowData.
It makes it easier to change the order of the fields or add more fields in the future.

like
`
rowData := []interface{}{
		ddlInfo.SchemaVer,
		ddlOwner,
		ddlJob,
		bgInfo.SchemaVer,
		bgOwner,
		bgJob,
	}
`